.github/workflows/deploy.yml

```
@@ -55,23 +61,51 @@ jobs:
+            fi
+            echo "TAGS_EOF"
+          } >> $GITHUB_OUTPUT
+
+          if [ "$IS_RELEASE" = "true" ]; then
+            echo "platforms=linux/amd64,linux/arm64,linux/arm/v7" >> $GITHUB_OUTPUT
+          else
+            echo "platforms=linux/arm64" >> $GITHUB_OUTPUT
+          fi
 
       - name: Build & push image
         uses: docker/build-push-action@v4
```

actionlint (1.7.12)

[error] the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)